### ISSUE TITLE
Test Results variable_cleanup_for_assemblies_adoc

### ISSUE THREAD

**Issue by taroth21** [tomschr/daps#672]
### Problem description
I cloned the doc-sleha repo and built the first example setup described in your mail, `category-setup`. The directory structure looks like this:

```
category-installation 
  images
  sources
  DC-SLE-HA-installation  
common-files
common-images 
``` 

Then I tried some commands as recommended in your mail, but they failed.

### Steps to reproduce problem

First command
```
✘-1 /virtual/doc-sleha/category-installation [daps-test L|✚ 187…3] 
16:56 $ gitdaps -d DC-SLE-HA-installation --srcdir sources --imgsrcdir images pdf
```

Result:
```
ERROR: Fatal: /virtual/doc-sleha/category-installation/images does not exist.
The IMG_SRC_DIR needs to be relative to /virtual/doc-sleha/category-installation and needs to exist.
```

Second command:
```
✘-1 /virtual/doc-sleha/category-installation [daps-test L|✚ 187…3] 
gitdaps -m sources/MAIN.SLEHA.xml --srcdir sources --imgsrcdir image
```

Result:
`ERROR: The options --main and --srcdir exclude each other.`

**Comment by janajaeger** [comment]
Can confirm the same behavior for the "other" directory setup as well.

**Comment by fsundermeyer** [comment]
Thanks, fixed with 4451329e
